src/app/pages/list/tree/tree.component.scss

```diff
@@ -71,12 +71,23 @@
             color: map-get($blui-blue, 500) !important;
         }
     }
+
```

Comment:
This sections shouldn't be needed; have you tried applying `color="primary"` on the `<mat-radio-button>`?

Reply:
color="primary" worked. Thanks!!